keys/ecc_test.go

```diff
@@ -14,6 +14,9 @@ func TestECCPasses(t *testing.T) {
 
 	checks := []ECC{
```

Comment:
This list doesn't change between TestECCFails and TestECCPasses, could we instead lift it out to make it a variable that will only need to be changed once?